blivet/populator/populator.py

```diff
@@ -64,16 +64,12 @@ def parted_exn_handler(exn_type, exn_options, exn_msg):
 
 
 class PopulatorMixin(object, metaclass=SynchronizedMeta):
-    def __init__(self, passphrase=None, luks_dict=None, disk_images=None):
+    def __init__(self, disk_images=None):
         """
-            :keyword passphrase: default LUKS passphrase
-            :keyword luks_dict: a dict with UUID keys and passphrase values
-            :type luks_dict: dict
             :keyword disk_images: dictoinary of disk images
             :type list: dict
-
         """
-        self.reset(passphrase=passphrase, luks_dict=luks_dict, disk_images=disk_images)
+        self.reset(passphrase=None, luks_dict={}, disk_images=disk_images)
 
     def reset(self, passphrase=None, luks_dict=None, disk_images=None):
```

Comment:
The original idea was to keep those so calling reset can set passphrase/luks_dict without additional method call. Removed anyway, since apparently it is not needed so often.